pkgs/top-level/all-packages.nix

```diff
@@ -14539,4 +14539,8 @@ with pkgs;
   davis = callPackage ../by-name/da/davis/package.nix {
     php = php83; # https://github.com/tchapi/davis/issues/195
   };
+
+  gpac-unstable = callPackage ../by-name/gp/gpac/package.nix {
```

Comment:
Please try to avoid appending to all-packages to avoid possible merge conflicts.